.github/workflows/build.yml

```diff
@@ -173,3 +173,23 @@ jobs:
         run: cmake --build build/ --config Release
       - name: "test: libterminal"
         run: .\build\src\libunicode\Release\unicode_test.exe
+
+
+  Fedora:
+    name: Fedora
+    runs-on: ubuntu-22.04
+    container: fedora:rawhide
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install build dependencies
```

Comment:
better to use script from `scripts/install-deps.sh`

Reply:
There are not many system dependencies when building this on Fedora. So I think it is not need to use install-deps.sh file?

Reply:
If we include the 'install-deps.sh' script in  actions, we can ensure that everything is up to date. so users won't face any issues when installing libunicode on Fedora.